src/Activity.js

```diff
@@ -7,51 +7,31 @@ class Activity extends StatsParent {
     this.user = user;
   }
 
-  findUser() {
-    return this.activityData.filter(person => person.userID === this.user.id);
-  }
-
-  returnWeekOfData(week, userData) {
-    return [...userData].splice((-7 * week), 7);
-  }
-
-  returnWeek(week) {
-    var specificUser = this.findUser()
-    return [...specificUser].splice(-7 * week, 7).map(day => day.date);
-  }
-  // returnUserDataForDay(date, relevantProperty) {
-  //   let specificUser = this.findUser();
-  //   return specificUser.find(day => day.date === date)[relevantProperty];
-  // }
-
-  returnMilesWalked() {
-    let specificUser = this.findUser();
+  returnMilesWalked(id) {
+    let specificUser = this.findUser(this.activityData, id);
```

Comment:
We set this variable in a lot of the methods I'm wondering if it would be worth it to make this a property of the class?  In my FitLit project I just chained off the method call, an example would be for metStepGoal on line 21
```this.findUser(this.activityData, id).find(day => day.date === date).numSteps
      return numSteps >= this.user.dailyStepGoal```
Definitely not a need to fix but a thought I'd love you hear the team's opinion on.